packages/graphql-config-utilities/CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ and adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ### Changed
 
+- Reorder exports map to prioritize the `esnext` condition. [[#2148](https://github.com/Shopify/quilt/pull/2148)]
 - Clean test output. [[#2091](https://github.com/Shopify/quilt/pull/2091)]
```

Comment:
This hasn't been released, right?

Reply:
Looks like it hasn't - the last change was just in test files, which are ignored when [determining what to release](https://github.com/Shopify/quilt/blob/main/lerna.json#L7)